lib/galaxy/visualization/plugins/plugin.py

```diff
@@ -71,10 +71,10 @@ def __init__(self, app, path, name, config, context=None, **kwargs):
         self.name = name
         self.config = config
         base_url = context.get('base_url', '')
-        self.base_url = '/'.join([base_url, self.name]) if base_url else self.name
+        self.base_url = '/'.join((base_url, self.name)) if base_url else self.name
```

Comment:
Maybe also remove the join in favor of a pure string concat, as you did below?

Reply:
In general I think the join is clearer when there are multiple variables to join using a separator, like in this case.
In the case below there was only one variable really, so it seemed better to just append to it using `+`.
Hopefully that makes sense?

Reply:
:+1: